src/recipe.js

```diff
@@ -16,6 +16,13 @@ class Recipe {
     return recipeIngredients;
   }
 
+  returnIngredientId(ingredientData, ingredientName) {
+    const ingredient = ingredientData.find(ingredient => {
+      return ingredient.name === ingredientName;
+    });
+    return ingredient.id;
+  }
+
```

Comment:
We would need to remove this method now yeah? Since it was created in the IngredientRepo class?

Reply:
Hmmm, I think you're right. I don't see a use for it.